lib/tasks/courses_wikis.rake

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+namespace :courses_wikis do
+  desc 'Creates CoursesWikis association for existing courses through revisions'
+  task migrate: :environment do
+    Rails.logger.debug 'Creating CoursesWikis associations'
+    Course.all.each do |course|
+      home_wiki_id = course.home_wiki.id # Home Wiki is already tracked
+      wiki_ids = course.revisions.pluck(Arel.sql('DISTINCT wiki_id')).uniq - [home_wiki_id]
```

Comment:
I think this should be done with the same algorithm we use already for wiki_ids: home_wiki, plus the wikis for the course's assignments.

Reply:
I had modified wiki_ids implementation that time so used the code from it and I didn't wanted home_wiki to be a part of courses_wikis. Now all this changes and I will edit it so.